src/time_integration/paired_explicit_runge_kutta/methods_PERK3.jl

```diff
@@ -0,0 +1,338 @@
+using DelimitedFiles: readdlm
+
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# Initialize Butcher array abscissae c for PairedExplicitRK3 based on SSPRK33 base method
+function compute_c_coeffs(num_stages, cS2)
+    c = zeros(num_stages)
+
+    # Last timesteps as for SSPRK33, see motivation in Section 3.3 of
+    # https://doi.org/10.1016/j.jcp.2024.113223
+    c[num_stages - 1] = 1.0f0
+    c[num_stages] = 0.5f0
```

Comment:
Hm, this is somewhat inconsistent, as `zeros` will create a `Float64` array by default. It is probably better to use the type of `cS2` to create zeros of that type and then use for one just `1` and for `0.5` maybe the `convert` function.